src/common/Pagination/index.js

```diff
@@ -12,29 +14,34 @@ import {
 } from "./styled";
 
 const Pagination = () => {
-  const [currentPage, setCurrentPage] = useState(1);
+  const dispatch = useDispatch();
+  const [searchParams, setSearchParams] = useSearchParams();
+  const currentPage = Number(searchParams.get("page")) || 1;
+  const totalPages = 500;
```

Comment:
How about limit it for max 500 ( API max ) but still geting proper value of there were less ??

Reply:
What do you mean

Reply:
It's now a fixed value, right?
But there will be some cases, when we will get an API response with less than 500 pages of results.
...
ok, mostly in search results, so we can update it with search abbility.